packages/elasticsearch-client/src/indexDocumentBulk.ts

```diff
@@ -37,10 +44,18 @@ export const indexDocumentBulk = async <T>({
   indexConfig,
   documents,
 }: IndexDocumentBulkParams<T>): Promise<IndexDocumentBulkResponse> => {
-  const body = documents.flatMap(({ id, document }) => [
-    { index: { _index: index, _id: id } },
-    indexConfig.convertToIndexDocument(document),
-  ]);
+  const body: BulkRequest['operations'] = documents.flatMap(
```

Comment:
Could you speak to you thinking around the decision to add multiple actions to the `indexDocumentBulk` function rather than creating new `deleteDocumentBulk` and `deleteDocument` functions?

Reply:
Ohhh... I see. We are mixing types of bulk operations into a single call. That makes sense. Perhaps we should change the file/function naming a bit to reflect that? `index` is definitely meant to be the verb in the function name here. Would something like `executeBulk()` be less confusing in the future?

Reply:
Yes, good point, I'll update that. I was thinking of 'indexing' in the broad sense of 'indexing our resources' rather than a specific ElasticSearch index operation, but what you suggest is way less ambiguous - especially since it's in the ElasticSearch client